lib/jsdom/living/nodes/HTMLAnchorElement-impl.js

```diff
@@ -2,14 +2,41 @@
 const idlUtils = require("../generated/utils");
 const HTMLElementImpl = require("./HTMLElement-impl").implementation;
 const HTMLHyperlinkElementUtilsImpl = require("./HTMLHyperlinkElementUtils-impl").implementation;
+const {navigate} = require("../window/navigation");
+const isDisabled = require("../helpers/form-controls").isDisabled;
 
+// https://html.spec.whatwg.org/#the-a-element
 class HTMLAnchorElementImpl extends HTMLElementImpl {
   constructor(args, privateData) {
     super(args, privateData);
 
     this._htmlHyperlinkElementUtilsSetup();
   }
 
+  _activationBehavior() {
+    if (this.hasAttribute("disabled")) {
+      return;
+    }
+
+    if (!this.href || !this.url) {
+      return;
+    }
+
+    const document = this._ownerDocument;
+    if (!document) {
+      return;
+    }
```

Comment:
I don't believe this is ever possible. Maybe you are thinking of the fully active check, but that is a more complex beast, probably best left as a TODO.